### PR TITLE
Add files

### PR DESCRIPTION
**Pull request by pankominek**

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+
+const checkButton = document.querySelector('#checkCard');
+
+const checkNumber = number => {
+    const cardNumbers = number.split('');
+
+    let even = cardNumbers
```
Comment (IwoHerka) [inline js/scripts.js:7]: Can be `const`.

File: js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        .map(data => data - 9);
+
+    const evenSingleDigit = even
+        .filter(element => element <= 9);
+
+    const formatedNumbers = evenSingleDigit
+        .concat(evenTwoDigit)
+        .concat(odd);
+
+    let sumNumbers = 0;
```
Comment (IwoHerka) [inline js/scripts.js:26]: Can be `const` too. How would you use reduce to calculate `sumNumbers` in place?

File: js/scripts.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    return sumNumbers % 10 === 0;
+}
+
+const displayResult = (result, cardNumber) => {
+    const sectionNumberCard = document.querySelector('#sectionNumberCard');
+    sectionNumberCard.insertAdjacentHTML('afterbegin', `<span class='card__output' >${cardNumber}</span>`);
+    
+    const displayCardNumber = document.querySelector('.card__output');
+
+    result
```
Comment (IwoHerka) [inline js/scripts.js:39]: Use ternary only if returning a result. Convert this to standard `if` for readability.

File: js/scripts.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        .concat(odd);
+
+    let sumNumbers = 0;
+
+    formatedNumbers.forEach(number => sumNumbers += number);
+
+    return sumNumbers % 10 === 0;
+}
+
+const displayResult = (result, cardNumber) => {
```
Comment (IwoHerka) [inline js/scripts.js:33]: `result` could be named to something more readable like `isValid`.

File: js/scripts.js (hunk 1 above)
Comment (pankominek) [reply js/scripts.js:7]: ok

File: js/scripts.js (hunk 2 above)
Comment (pankominek) [reply js/scripts.js:26]: `    const sumNumbers = formatedNumbers.reduce((a, b) => a + b);`

File: js/scripts.js (hunk 4 above)
Comment (pankominek) [reply js/scripts.js:33]: ok

File: js/scripts.js (hunk 3 above)
Comment (pankominek) [reply js/scripts.js:39]: ok

File: css/main.css [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+.card__button {
+    padding: 10px;
+    cursor: pointer;
+    text-transform: uppercase;
+    font-style: italic;
+}
+
+.card__button:hover {
+    color: blue;
```
Comment (dbisiakowski) [inline css/main.css:77]: Use hex.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Luhn algorithm</title>
+        <link rel="stylesheet" type="text/css" href="css/main.css">
+    </head>
+    <body>
+        <main class="card">
+                <header class="card__header">
```
Comment (dbisiakowski) [inline index.html:12]: The header should be above the main tag
https://html.spec.whatwg.org/multipage/grouping-content.html#the-main-element

File: index.html [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+                    <h1 class="card__title">Makimo bank polska</h1>
+                </header>
+                <section id ="sectionCardNumber" class="card__number-section number">
+                    <label id="labelCardNumber" class="number__label">
+                        <span id="spanCardNumber" class="number__desc">Enter the 16 digit credit card number...</span>
+                        <input id="inputCardNumber" class="number__input" type="number" value="4113601974804222" placeholder="XXXX XXXX XXXX XXXX">
+                    </label>
+                </section>
+                <footer class="card__footer">
```
Comment (dbisiakowski) [inline index.html:21]: The footer should be below the main tag
https://html.spec.whatwg.org/multipage/grouping-content.html#the-main-element

File: js/scripts.js [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+    const labelInputCardNumber = document.querySelector('#spanCardNumber');
+    labelInputCardNumber.textContent = 'Your card number is too short';
+}
+
+const verifyCardNumber = () => {
+    const GOOD_LENGTH = 16;
+    const inputCardNumber = document.querySelector('#inputCardNumber');
+    const cardNumber = getCardNumber(inputCardNumber);
+
+    if (!cardNumber || cardNumber.length !== GOOD_LENGTH) {
```
Comment (dbisiakowski) [inline js/scripts.js:57]: If you enter more than 16 digits you got wrong information. Instead "Your card number is too long" you got "Your card number is too short"

File: js/scripts.js [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+        .filter(element => element <= 9);
+
+    const formatedNumbers = evenSingleDigit
+        .concat(evenTwoDigit)
+        .concat(odd);
+
+    const sumDigits = formatedNumbers.reduce((prevDigit, digit) => prevDigit + digit);
+
+    return sumDigits % 10 === 0;
+}
```
Comment (dbisiakowski) [inline js/scripts.js:30]: Arrow functions are variables so add semicolon on the end.

File: css/main.css (hunk 5 above)
Comment (pankominek) [reply css/main.css:77]: ok

File: js/scripts.js (hunk 8 above)
Comment (pankominek) [reply js/scripts.js:57]: hahaha. thanks :)

File: js/scripts.js (hunk 9 above)
Comment (InternMakimo) [reply js/scripts.js:30]: ok

Comment (pankominek) [reply js/scripts.js:30]: ok

File: index.html (hunk 6 above)
Comment (pankominek) [reply index.html:12]: I added more html code.

**Pull request merged**